CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog][format], and this project adheres to
 
 ## Unreleased
 
+### Added
+* A new configuration option `g:cmake_statusline` controls whether Vim-CMake
+  will override the |statusline| option for the CMake console window.
+
```

Comment:
We should add a note about `cmake#Get()` here as well

Reply:
Planned as part of documentation. Will do